### ISSUE TITLE
thyroid datasets don't match with literature

### ISSUE THREAD

**Issue by amueller** [openml/OpenML#675]
I'm trying to understand the UCI thryoid dataset(s?):
https://www.openml.org/search?q=thyroid&type=data

I can't consolidate the data on openml with the data on UCI and the description. thyroid should have 6832 or 5473 samples depending on the version, none of the datasets on OpenML has that.
There is a standard split into training and test set, and it looks like OpenML only has the training sets, but I'm not sure.
This dataset pretty explicitly gives the number of training samples, so this is missing the test set and the correct split:
https://www.openml.org/d/40497
